### PR TITLE
docs(aio): add short description for entryComponents

### PR DESCRIPTION
**Pull request by VadimDez**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[x] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by trotyl** [comment]
Relates to #21257

File: aio/content/guide/cheatsheet.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ is available to <code>declarations</code> of this module.</p>
 <td><p>List of dependency injection providers visible both to the contents of this module and to importers of this module.</p>
 </td>
 </tr><tr>
+<td><code><b>entryComponents:</b> [SomeComponent, OtherComponent]</code></td>
+<td><p>List of components that will be compiled when the module is defined.</p></td>
```
Comment (gkalpak) [inline aio/content/guide/cheatsheet.md]: I don't think this description is accurate. They are components that _could_ be created and attached to nodes without being part of another components template (i.e. dynamically), but that doesn't mean they _will be_ compiled/created.

**Review by kapunahelewong — APPROVED** [review]
Thank you, this is great! (PS: you might like to rebase as important changes have been merged. And squash.)

**Review by brandonroberts — APPROVED** [review]

**Comment by mlc-mlapis** [comment]
@VadimDez ... and also not referenced in any defined routes in that module.

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**